__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const data = require("../db/data/test-data")
+const seed = require("../db/seeds/seed")
+const db = require("../db/connection")
+const app = require("../app")
+const request = require("supertest")
+
+beforeAll(() => seed(data))
+afterAll(() => db.end())
+
+describe("GET/topics", () => {
+    test("200: Should respond with all topics in an array of objects containing the properties slug and description", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const topics = body;
+          topics.forEach((topic) => {
+            expect(typeof topic.slug).toBe("string");
+            expect(typeof topic.description).toBe("string");
+            expect(Object.keys(topic).length).toBe(2)
+          });
+        });
```

Comment:
for all tests where we use a forEach loop we need to make sure that we're actually getting the right number of items back. E.g. if the endpoint was returning no items instead of our expected arr, then it would still pass the test because the forEach wouldn't run and no assertions would fail. 

So what kind of check could go alongside our forEach?